python/setup.py.in

```diff
@@ -0,0 +1,9 @@
+#!@PYTHON@
+
+from setuptools import setup
+
+setup(name='pacemaker',
+      version='@PACKAGE_VERSION@',
```

Comment:
[Here](https://setuptools.pypa.io/en/latest/userguide/distribution.html) is what setuptools accepts. It looks like maybe it doesn't like dots coming after a dash. Maybe we could just strip everything starting from the first non-alphanumeric character after the first dash (if any). That should leave the packaging number and just strip any commit ID, which may not be ideal from a versioning perspective but should be acceptable.

Reply:
Fixed, I think.  We'll find out on the next run through CI.